### PR TITLE
fix: get the correct terminal instance by terminal short id 

### LINKED ISSUE #1158: [BUG] 插件 API Terminal sendText 不生效
**描述你的问题（Describe the bug）**

发现 普通的 terminal 调用 sendText 正常，如果当前是 Javascript Debug Terminal 就不会生效。

**复现路径（To Reproduce）**

复现如 #1107 或者看 gif

**预期表现（Expected behavior）**

**问题截图（Screenshots）**
screenshort_20220601-143610

**环境信息（Environment）**

- OS: macOS
- Browser: chrome
- OpenSumi Version: main 分支

**补充信息（Additional context）**

### PR DESCRIPTION
**Pull request by erha19**
### Types

- [x] 🐛 Bug Fixes

### Background or solution

close #1158 

https://user-images.githubusercontent.com/9823838/171438764-8f08bc54-b244-4797-a095-90ae68b927d4.mp4

### Changelog

get the correct terminal instance by terminal short id

### CODE REVIEW

File: packages/terminal-next/src/browser/terminal.api.ts [hunk 1]
```diff
@@ -40,8 +40,6 @@ export class TerminalApiService implements ITerminalApiService {
   @Autowired(ITerminalNetwork)
   protected readonly network: ITerminalNetwork;
 
-  protected _entries = new Map<string, ITerminalExternalClient>();
```
Comment (erha19) [inline packages/terminal-next/src/browser/terminal.api.ts:43]: 移除了这里多余的 _entries 数据结构

File: packages/extension/src/browser/vscode/api/main.thread.terminal.ts [hunk 2]
```diff
@@ -194,6 +207,10 @@ export class MainThreadTerminal implements IMainThreadTerminal {
   }
 
   private _getTerminalProcess(terminalId: string): ITerminalProcessExtHostProxy {
+    if (this.shortId2LongIdMap.has(terminalId)) {
```
Comment (bytemain) [inline packages/extension/src/browser/vscode/api/main.thread.terminal.ts:210]: 统一用 transform 这个方法吧

**Comment by bytemain** [comment]
this close: https://github.com/opensumi/core/issues/1125

File: packages/extension/src/browser/vscode/api/main.thread.terminal.ts [hunk 3]
```diff
@@ -194,6 +207,9 @@ export class MainThreadTerminal implements IMainThreadTerminal {
   }
 
   private _getTerminalProcess(terminalId: string): ITerminalProcessExtHostProxy {
+    this.transform(terminalId, (id) => {
```
Comment (bytemain) [inline packages/extension/src/browser/vscode/api/main.thread.terminal.ts:210]: .....我来改一下这里

**Review by bytemain — APPROVED** [review]

**Pull request merged**